deployer/README.md

```diff
@@ -28,19 +28,23 @@ with regards to configuring AWS access credentials.
 ## Uploads
 
 The `poetry run deployer upload` command uploads files as well as redirects
-into an existing S3 bucket. As input, it takes a directory which contains the
-files that should be uploaded, but it also needs to know where to find
-any redirects that should be uploaded. By default it searches for redirects
-within the content directories specified by `--content-root` (or
-`CONTENT_ROOT`) and `--content-translated-root` (or `CONTENT_TRANSLATED_ROOT`).
+into an existing S3 bucket. Currently, we have three main S3 buckets that we
+upload into: `mdn-content-dev` (for variations or experimental versions of
+the site), `mdn-content-stage`, and `mdn-content-prod`.
+
+As input, the `poetry run deployer upload` command takes a directory which
```

Comment:
```suggestion
As input, the `upload` command takes a directory which
```
so you don't need to repeat the whole command again. Especially if we need to change it in two places some day.